src/js/components/List/__tests__/List-test.js

```diff
@@ -1,13 +1,153 @@
 import React from 'react';
 import 'jest-styled-components';
-import { cleanup, render, fireEvent } from '@testing-library/react';
+import 'jest-axe/extend-expect';
+import 'regenerator-runtime/runtime';
 
+import { cleanup, render, fireEvent } from '@testing-library/react';
+import { axe } from 'jest-axe';
 import { Grommet } from '../../Grommet';
 import { List } from '..';
 
 describe('List', () => {
   afterEach(cleanup);
 
+  test('list should have no violations', async () => {
+    const { container } = render(
+      <Grommet>
+        <List />
```

Comment:
May want to add some dummy data in the list here

Reply:
Went back and added data to the jest-axe test